modules/installation-and-upgrade/pages/proxy-container-setup.adoc

```diff
@@ -31,6 +32,8 @@ There are two ways how to generate {productname} configuration: use web interfac
 . In the [guimenu]``SSL certificate`` choose if new server certificate should be generated for {productname} Proxy or existing should be used.
 +
 Depending on the choice then provide either path to signing CA certificate to generate new certificate or path to server certificate and its key to be used as proxy certificate.
++
+The CA certificates on the server are stored in the [path]``/root/ssl-build`` directory.
```

Comment:
Could we state that this is only the generated CA files and not user-provided ones?

Reply:
i hope it is better now, I'll now check Uemit's feedback provided via bugzilla.